config/environments/sandbox.rb

```diff
@@ -1,14 +1,14 @@
 require Rails.root.join("config/environments/production")
 
 Rails.application.configure do
-  config.log_level = :info
+  config.log_level = :debug
   config.ssl_options = { redirect: { exclude: ->(request) { request.path.include?("/healthcheck") } } }
 
   # Enable/disable aspects of the NPQ separation
   config.npq_separation = {
-    admin_portal_enabled: false,
-    api_enabled: false,
+    admin_portal_enabled: true,
+    api_enabled: true,
```

Comment:
I thought we wanted to test on separation instead as it has all the setup? not keen on turning this on on sandbox as its connected to the ECF sandbox 🤔